### PR TITLE
We no longer support TPR, CRD is always enabled

### PR DESCRIPTION
**Pull request by mortenlj**
We removed support for TPR from fiaas-deploy-daemon, so no need to support it in Skipper anymore.

### CODE REVIEW

File: fiaas_skipper/config.py [hunk 1]
```diff
@@ -52,10 +52,6 @@ def _parse_args(self, args):
                             action="store_true")
         parser.add_argument("--port", help="Port to use for the web-interface (default: %(default)s)", type=int,
                             default=5000)
-        parser.add_argument("--enable-tpr-support", help="Enable Third Party Resource support.",
-                            action="store_true")
-        parser.add_argument("--enable-crd-support", help="Enable Custom Resource Definition support.",
-                            action="store_true")
```
Comment (oyvindio) [inline fiaas_skipper/config.py]: These flags are used in the helm chart, so they probably have to be removed from there as well, along with the associated values: https://github.com/fiaas/skipper/blob/a19f9df2fa78c2201101b2f736281c65d6130331/helm/fiaas-skipper/templates/config_map.yaml#L28-L29

Comment (mortenlj) [reply fiaas_skipper/config.py]: Good catch. Should be fixed now.

File: helm/fiaas-skipper/Chart.yaml [hunk 2]
```diff
@@ -17,4 +17,4 @@ apiVersion: v1
 description: Skipper controls deployment and updates of FIAAS components
 name: fiaas-skipper
 home: https://github.com/fiaas/skipper
-version: 0.1.0
```
Comment (birgirst) [inline helm/fiaas-skipper/Chart.yaml]: Updating the minor version is not needed and will be added by the deploy scripts.

Comment (mortenlj) [reply helm/fiaas-skipper/Chart.yaml]: Reverted that change.

**Comment by mortenlj** [comment]
> Remove bumping the chart version. It is in sync with the tag that will be created automatically when merging this pr.

Done.

**Pull request merged**